### PR TITLE
Development

### PR DESCRIPTION
**Pull request by rajbalaji003**
Add style files and html to  Development

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,42 @@
...
 </head>
 <body>
-    
+    <header>
+        <nav class="menu">
+            <div>
+                <a href="#" class="logo">My Logo</a>
+            </div>
+            <div class="menu-icon">
+                <img src="/img/Icon - Menu.png" alt="">
```
Comment (christianbiring1) [inline index.html:23]: Kindly make sure there is no space on the name of your images that's why you can see they are not displayed on your project.
`hint`: `You can separate the name with an hyphen or underscore not a generic space` 
Screenshot from 2022-09-21 19-06-56

File: index.html [hunk 2]
```diff
@@ -4,9 +4,42 @@
...
+                <p class="my-name-sub">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a href="#" class="connect-me">LET'S CONNECT</a>
+            </div>
+            <div class="social-media-icon">
+                <img src="/img/social icons/twiter icon.svg" alt="">
+                <img src="/img/social icons/github icon.svg" alt="">
+                <img src="/img/social icons/icon.svg" alt="">
+                <img src="/img/social icons/Linkedin icon.png" alt="">
+                <img src="/img/social icons/medium icon.svg" alt="">
+            </div>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure your social links are inside and unordered list `ul` and not a `div`
- [ ] And also grab all the `img` inside a `anchor <a></a>` as they are links to your social media and each one of them inside a `li`  
`<ul>`
`<li> <a href="'> </a></li>`
`</ul>`

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  --primary-color: #6070ff;
+  --secondary-color: #344563;
+  --tertiary-color: #396df2;
+  --main-color: #172b4d;
+  --whitelike-color: #ebebff;
+  --neutral-color: #7a869a;
+  --general-color: #f7f7f9;
+}
+
+@media only screen and (max-width: 750px) {
```
Comment (christianbiring1) [inline css/style.css:18]: [OPTIONAL]
Since you are going with the mobile-first approach, I think you do not need this media query right here.

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @rajbalaji003 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
:heavy_check_mark: All the linters are passing
:heavy_check_mark: You have used the `FlexBox` to display the element on the page

## Required Changes ♻️
- [ ] Kindly make sure you add a descriptive summary to your PR saying what the project is about. `hint: You can use the project description`
- [ ] Make sure your project is documented in a professional way. `you can use the microverse README template and customize it in your project` 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jasonfelice — APPROVED** [review]
Hi @rajbalaji003, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**